packages/core/src/services/scales-cartesian.ts

```diff
@@ -304,14 +317,7 @@ export class CartesianScales extends Service {
 			];
 		}
 
-		// TODO - Work with design to improve logic
-		domain[1] = domain[1] * 1.1;
-
-		// if the lower bound of the domain is less than 0, we want to add padding
-		if (domain[0] < 0) {
-			domain[0] = domain[0] * 1.1;
-		}
-		return domain;
+		return addPaddingInDomain(domain, 10);
```

Comment:
What's `10`? Shouldn't that be configurable and stored somewhere in `Configuration.ts`?

Reply:
Added it as `paddingRatio` into axis configuration.